2024-12-07 05:30:22,455 - ERROR - Неверный тип данных для объекта Runner: Имя может быть только строкой, передано int
Traceback (most recent call last):
  File "C:\python urban\tests_12_4.py", line 36, in test_run
    runner = Runner(123, speed=10)
             ^^^^^^^^^^^^^^^^^^^^^
  File "C:\python urban\module_12_4.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
2024-12-07 05:30:22,456 - ERROR - Неверная скорость для Runner: Скорость не может быть отрицательной, сейчас -5
Traceback (most recent call last):
  File "C:\python urban\tests_12_4.py", line 23, in test_walk
    runner = Runner("Вася", speed=-5)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\python urban\module_12_4.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -5
